setup.py

```diff
@@ -80,9 +80,11 @@
       test_suite='plugin_tests',
       tests_require=test_requirements,
       ext_modules = cythonize(Extension(
-           "histomicstk.segmentation.label.isbf",
-           sources=["histomicstk/segmentation/label/isbf.pyx",
-                    "histomicstk/segmentation/label/isbfcpp.cpp"],
+           "histomicstk.segmentation.label.trace_object_boundary",
+           sources=[
+                "histomicstk/segmentation/label/trace_object_boundary.pyx",
+                "histomicstk/segmentation/label/trace_boundary_cpp.cpp"
+           ],
```

Comment:
Read the last part of [this](http://docs.cython.org/en/latest/src/userguide/wrapping_CPlusPlus.html#specify-c-language-in-setup-py).

Seems like the options and sources can also be passed directly from the .pyx file as follows. This will make setup.py simpler and also we wont have to change the setup.py file each time we cythonize a new function. 

Here is a simplified setup.py file:

```python
from distutils.core import setup
from Cython.Build import cythonize

setup(
    name = "rectangleapp",
    ext_modules = cythonize('*.pyx'),
)
```
And in the .pyx source file, write this into the first comment block, before any source code, to compile it in C++ mode and link it statically against the Rectangle.cpp code file:

```python
# distutils: language = c++
# distutils: sources = Rectangle.cpp
```